### ISSUE TITLE
Convert tree-sitter to Lezer trees for CodeMirror 6 support

### ISSUE THREAD

**Issue by ryyppy** [nkrkv/tree-sitter-rescript#196]
I always wanted to upgrade the ReScript online playground to the newest CodeMirror version, but CM transitioned from regex based language parsers to a more sophisticated parser system called [Lezer](https://lezer.codemirror.net/).

If the tree-sitter is stable enough to reflect all parts of the language, I thought it would be interesting to try to use the tree-sitter parser as a shim until I figured out how to proceed writing a dedicated Lezer parser.

https://discuss.codemirror.net/t/mapping-tree-sitter-trees-to-lezer-trees/2362

**Comment by nkrkv** [comment]
Yes, the grammar is stable enough and covers most quirky cases, so anyone can try to port to Lezer.

With this issue, are we trying to summon someone to do this? :)

**Comment by ryyppy** [comment]
Yeah, I kinda put this note here in case someone feels inspired to try it.
